components/admin/modules/SideBar.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react'
+import _ from 'lodash'
+import { Module } from '../../../graphql/index'
+import styles from '../../../scss/sideBar.module.scss'
+import { XIcon, ChevronRightIcon } from '@primer/octicons-react'
+
+type SideBarProps = {
+  modules: Module[] | []
+  selectedModule: number
+}
+export const SideBar: React.FC<SideBarProps> = ({
+  modules,
+  selectedModule
+}) => {
+  const moduleListData = [_.cloneDeep(modules) || []]
+  const lessonList = moduleListData[0].map((obj: any, arrIndex: number) => (
```

Comment:
Using `any` here defeats the purpose of using typescript -- please use the correct type definition